src/app/Hooks/HooksPage.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import {
+  Card,
+  PageSection,
+  CardBody,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+  Title,
+  Button,
+} from '@patternfly/react-core';
+import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+
+import {
+  useHasSufficientProvidersQuery,
+  useHooksQuery,
+  useClusterProvidersQuery,
+} from '@app/queries';
+
+import HooksTable from './components/HooksTable';
+import { ResolvedQueries } from '@app/common/components/ResolvedQuery';
+import { IHook } from '@app/queries/types';
+import AddEditHookModal from './components/AddEditHookModal';
+
+const HooksPage: React.FunctionComponent = () => {
+  const hooksQuery = useHooksQuery();
+
+  const [isAddEditModalOpen, toggleAddEditModal] = React.useReducer((isOpen) => !isOpen, false);
+  const [hookBeingEdited, setHookBeingEdited] = React.useState<IHook | null>(null);
+
+  const toggleModalAndResetEdit = () => {
+    setHookBeingEdited(null);
+    toggleAddEditModal();
+  };
+
+  const openEditHookModal = (hook: IHook) => {
+    setHookBeingEdited(hook);
+    toggleAddEditModal();
+  };
+
+  return (
+    <>
+      <PageSection variant="light">
+        <Title headingLevel="h1">Hooks</Title>
+      </PageSection>
+      <PageSection>
+        <ResolvedQueries
+          results={[hooksQuery]}
+          errorTitles={[
+            'Error loading hooks',
+            'Error loading hooks from cluster',
+            'Error loading plans',
+          ]}
+          errorsInline={false}
+        >
```

Comment:
@gildub I'm getting errors trying to load the page because the length of `results` and `errorTitles` is not the same here. Makes me think I should have changed the props API for ResolvedQueries so TS could catch this.. but can you change it so there is only one `errorTitle` per `result`? (for now, just `Error loading hooks`)

![Screen Shot 2021-05-05 at 8 51 13 AM](https://user-images.githubusercontent.com/811963/117143670-0f0e9d80-ad7f-11eb-8c27-6a542e1674a9.png)

Reply:
@mturley, sure, done.

Although we'll need the plans to be retrieved to find out any dependencies with the hooks.